301-azure-relay-create-authrule-namespace-and-hybridconnection/README.md

```diff
@@ -1,7 +1,12 @@
 # Create an Azure Relay namespace & it's entities with SAS Policies
 
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F301-azurerelay-create-authrule-namespace-and-hybridconnection%2Fazuredeploy.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F301-hdinsight-windows-storm-iothub%2Fazuredeploy.json" target="_blank">
```

Comment:
why are we pointing the README for 301-azure-relay-... to the template for 301-hdinsight-windows-...?